internal/cache/controller_obj_cache.go

```diff
@@ -2431,8 +2457,70 @@ func checkSegroupLabels(client *clients.AviClient) bool {
 	return true
 }
 
-func checkTenant(client *clients.AviClient) bool {
+// RemoveSeGroupLabels removes labels on the SeGroup
+func RemoveSeGroupLabels(client *clients.AviClient, seGroup *models.ServiceEngineGroup) bool {
```

Comment:
This behaviour needs to be validated with the SE team, If the existing routes are there for the SEG then those will not be removed from SE if we were to delete/change the labels.